app/assets/stylesheets/ama_layout/layout_components/progress-bar.scss

```diff
@@ -32,6 +32,10 @@
     color: $white;
     overflow: hidden;
     padding: 6px 0;
+
+    &:last-child:not(:first-child){
+      float: none;
```

Comment:
yes, this is the fix for that single pixel that was showing on the progress tracker in the last step.